source/support.tex

```diff
@@ -4623,183 +4569,410 @@
 \end{note}
 \end{itemdescr}
 
-\rSec2[cmp.alg]{Comparison algorithms}
+\rSec2[cmp.concept]{Concept \tcode{ThreeWayComparable}}
 
-\indexlibrary{\idxcode{strong_order}}%
-\begin{itemdecl}
-template<class T> constexpr strong_ordering strong_order(const T& a, const T& b);
-\end{itemdecl}
+\begin{codeblock}
+template <typename T, typename Cat>
+  concept @\placeholder{compares-as}@ =                 // \expos
+    Same<common_comparison_category_t<T, Cat>, Cat>;
+
+template<class T, class U>
+  concept @\placeholder{partially-ordered-with}@ =      // \expos
+    requires(const remove_reference_t<T>& t, const remove_reference_t<U>& u) {
+      { t < u } -> Boolean;
+      { t > u } -> Boolean;
+      { t <= u } -> Boolean;
+      { t >= u } -> Boolean;
+      { u < t } -> Boolean;
+      { u > t } -> Boolean;
```

Comment:
A space should come after the operator for this and the other 3 lines using `<` and `>`, so that they be aligned like in `StrictTotallyOrdered`.

Reply:
Fixed.